src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/RestoreOperation/RestoreDatabaseTaskDataObject.cs

```diff
@@ -680,6 +680,27 @@ private string CombineToServerConnectionPath(string root, string filePath)
             return result;
         }
 
+        private string GetDirectoryName(string filePath) 
```

Comment:
should this be a helpwe in the PathWrapper class so we can share logic elsewhere?